app/main.py

```diff
@@ -1 +1,17 @@
-# write your code here
+from app.errors import VaccineError, NotWearingMaskError
+from app.cafe import Cafe
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
```

Comment:
Indicate list of what instances you expect to obtain for `friends` in the type annotation

Reply:
done